packages/ui/__tests__/components/Layout.test.tsx

```diff
@@ -9,7 +9,8 @@
 // WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 // PARTICULAR PURPOSE. See the GNU General Public License for more details.
 
-import { render, screen } from '@testing-library/react';
+import { waitFor, render, screen } from '@testing-library/react';
```

Comment:
Changes to this file can be ignore as they were already reviewed in [#34](https://github.com/cartesi/explorer/pull/34).